### ISSUE TITLE
2019-2020 and 2021-2022 options on election dropdown, but no data

### ISSUE THREAD

**Issue by JonellaCulmer** [fecgov/fec-cms#2085]
**What we're after:** On election profile pages, upcoming cycle information appears to be populated in the dropdown, but the data is not appearing. This is more likely for the 2019-2020 period than the 2021-2022 data, but neither can be confirmed by comparing to classic as that information is not currently listed. Currently, candidate data is only listed up until 2018. 

This may just be a matter of the front-end functioning, but nothing is set-up yet in the back-end to support it. 

**Completion criteria:**
- [ ] "Turn on" the back-end data to support front-end drop-down, or
- [ ] If not possible, remove options beyond 2017-2018 from the front-end if we don't yet have the back-end in a place to support it

Examples: 
<img width="751" alt="screen shot 2018-06-12 at 4 11 23 pm" src="https://user-images.githubusercontent.com/31663028/41314978-3fd63816-6e5c-11e8-9dac-7a6174419edd.png">
<img width="750" alt="screen shot 2018-06-12 at 4 11 33 pm" src="https://user-images.githubusercontent.com/31663028/41314979-3fe162ae-6e5c-11e8-858a-38dc399b1088.png">

**Comment by patphongs** [comment]
We decided to remove any cycle options greater than 2018 in the new election profile page implementation until we can get the 2020 data added in: https://github.com/fecgov/fec-cms/pull/1984

**Comment by AmyKort** [comment]
This issue is ultimately blocked by https://github.com/fecgov/openfec/issues/3254 . Should we postpone it until 6.6? Is the work to remove options beyond 2017-2018 complete?
